Tests/SonarScanner.MSBuild.Shim.Test/AdditionalFilesServiceTest.cs

```diff
@@ -179,8 +178,8 @@ public void AdditionalFiles_ExtensionsFound_AllExtensionPermutations(string prop
             "valid.JSX",
             "invalid.ajs",
             "invalidjs",
-            @"C:\.js",
-            @"C:\.jsx"
+            ".js",
+            ".jsx"
```

Comment:
```suggestion
            $"{TestUtils.DriveRoot()}.js",
            $"{TestUtils.DriveRoot()}.jsx"
```
Is there a reason to not do this?